samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -937,7 +912,10 @@ static void light_lightness_range_status(struct bt_mesh_model *model,
 					 struct bt_mesh_msg_ctx *ctx,
 					 struct net_buf_simple *buf)
 {
-	printk("Acknownledgement from LIGHT_LIGHTNESS_SRV (range_min) = %04x",
+	printk("Acknownledgement from LIGHT_LIGHTNESS_SRV (status code) = %02x",
+	       net_buf_simple_pull_u8(buf));
+
+	printk("  (range_min) = %04x",
```

Comment:
Looks like a missing `\n` at the end. Also, I think you can fit the entire function call on one line, without hitting the 80 character limit

Reply:
I'm considering it as single line. Hence add '\n' in last one.

Reply:
Or is the attention to put all three printk calls to a single line of output?

Reply:
ok, nevermind my comment then (except for the seemingly unnecessarily split function call)